crates/worktree/src/worktree_tests.rs

```diff
@@ -2712,6 +2712,28 @@ async fn test_propagate_git_statuses(cx: &mut TestAppContext) {
     );
 }
 
+#[gpui::test]
+async fn test_privacy(cx: &mut TestAppContext) {
```

Comment:
Very minor: I'd give this test a more specific name like `testprivate_single_file_worktree`